SA3/qa_web/templates/qa_web/answerspage.html

```diff
@@ -0,0 +1,55 @@
+
+{% load static %}
+<html>
+  <head>
+	<meta charset="UTF-8">
+	
+    <title>SA3 QA Website Answers</title>
+    <script src="{% static 'Markdown.Converter.min.js' %}"></script>
+    <script src="{% static 'Markdown.Editor.min.js' %}"></script>
+    <script src="{% static 'Markdown.Sanitizer.min.js'%}"></script>
+
+    <link rel="stylesheet" href="pagedown.css" />
+
+    <style>
+      .wmd-button > span {
+        background-image: url('wmd-buttons.svg');
+        background-repeat: no-repeat;
+        background-position: 0px 0px;
+        width: 20px;
+        height: 20px;
+        display: inline-block;
+      }
+      #wmd-input {
+        padding: 10px;
+        background-color: white;
+        max-width: 660px;
+        min-width: 660px;
+      }
+      .wmd-spacer {
+        background-color: white;
+        width: 0;
+      }
+      #wmd-button-bar {
+        height: 44px;
+        padding: 0 12px;
+        width: 660px;
+        border: 1px solid #c8ccd0;
+        box-sizing: border-box;
```

Comment:
Make sure the `css` is in a separate static file.

Reply:
Try to add the same theme CSS/ bootstrap template from the homepage.